race_test.go

```diff
@@ -0,0 +1,41 @@
+package smtpmock_test
+
+import (
+	"fmt"
+	"net/smtp"
+	"testing"
+	"time"
+
+	smtpmock "github.com/mocktools/go-smtp-mock/v2"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRace(t *testing.T) {
+	hostName := "localhost"
+	server := smtpmock.New(smtpmock.ConfigurationAttr{
+		HostAddress: hostName,
+	})
+
+	if err := server.Start(); err != nil {
+		t.Log(err)
+		t.FailNow()
+	}
+
+	go func() {
+		_ = smtp.SendMail(
```

Comment:
Can we use existent test helper (`runSuccessfulSMTPSession()`) instead of this direct call?